Source/MonoGame.Extended/Maps/Tiled/TiledMap.cs

```diff
@@ -41,6 +40,7 @@ public void Dispose()
         public int Height { get; }
         public int TileWidth { get; }
         public int TileHeight { get; }
+        public List<TiledTileset> Tilesets { get; }
```

Comment:
@craftworkgames Have you considered [`IReadOnlyList<T>`](https://msdn.microsoft.com/en-us/library/hh192385)?. `List<T>` implements the interface out of the box so could just cast `List<T>` to `IReadOnlyList<T>` and expose the latter in the API.